tasks/sonar_config.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+- name: 'create sonar database'
+  postgresql_db: name="{{ sonar_postgres_db_name }}" template='template0'
+                 encoding='UTF-8'
+                 #lc_collate='en_EN.UTF-8' lc_ctype='en_EN.UTF-8'
+  tags:
+    - sonar
+    - sonarconfig
+    - database
+    - postgresql
+
+- name: 'create sonar user'
+  postgresql_user: db=sonar name='{{ sonar_username }}' password='{{ sonar_password }}'
```

Comment:
Should it be `db='{{ sonar_postgres_db_name }}'`?

Reply:
ah yes